building-with-ocean/build-a-marketplace/forking-ocean-market.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Forking Ocean Market
+order: 1
+hideLanguageSelector: true
+description: Forking and running Ocean Market locally.
+featuredImage: images/creatures/mantaray/mantaray-full@2x.png
+---
+
+# Forking and running Ocean Market locally
+
+Have you ever thought about monetizing the data that your organization generates? You might imagine that this involves dealing with data brokers, but with Ocean you can sell your data directly to your customers with no third party in-between. Ocean makes this all super easy for you with some pretty cool tech under the hood.
+
+Using Ocean Market is already a big improvement on the alternatives that are out there, but it gets even better. Ocean Market is completely open-source and made freely available under the Apache 2 license. This means that you can fork Ocean Market and set up your own data marketplace in just a few steps. This guide will walk you through the process, you’ll be surprised how easy it is. No prior blockchain knowledge is required!
+
+
+- Fork Ocean Market
+
+- Clone the market locally
+
+- Install the dependencies
+
+- Run your Market fork for the first time
+
+## Fork Ocean Market
+
+The first step is to log into Github and navigate to https://github.com/oceanprotocol/market, you’ll need to log in or create a Github account. Now you need to click “Fork” in the top right-hand corner. If you are a member of an organization on Github, it will give you the option to clone it into either your personal account or the organization. Choose whichever is suitable for you.
```

Comment:
This link is not clickable. We can have something like: market [GitHub repository](https://github.com/oceanprotocol/market)